src/Bicep.Core/Semantics/Namespaces/SystemNamespaceType.cs

```diff
@@ -1524,6 +1527,11 @@ private static IEnumerable<NamespaceValue<Decorator>> GetSystemDecorators(IFeatu
         {
             static SyntaxBase SingleArgumentSelector(DecoratorSyntax decoratorSyntax) => decoratorSyntax.Arguments.Single().Expression;
 
+            static FunctionArgumentSyntax? GetArgumentByPosition(DecoratorSyntax decoratorSyntax, int position)
```

Comment:
>GetArgumentByPosition [](http://example.com/codeflow?start=43&length=21)

nit - Can you call this `TryGetArgumentByPosition()` to make it more obvious that you could get a null.